app/pagination.rb

```diff
@@ -28,19 +28,22 @@ def offset_from_page
   end
 
   def order_from_options(options)
-    order = options.fetch(:order, "-public_updated_at")
+    order_string = options.fetch(:order, '-public_updated_at')
 
-    if order.start_with?("-")
-      field = order[1..-1].to_sym
-      direction = :desc
-    else
-      field = order.to_sym
-      direction = :asc
-    end
-
-    raise_unless_valid_order_field(field)
+    orders = order_string.split(',').map(&:strip)
+    orders << 'id' if orders.none? { |o| o.match('[-]?id') }
 
-    { field => direction }
+    orders.each_with_object([]) do |order, result|
+      if order.start_with?("-")
+        field = order[1..-1].to_sym
+        direction = :desc
+      else
+        field = order.to_sym
+        direction = :asc
+      end
+      raise_unless_valid_order_field(field)
+      result << [field, direction]
```

Comment:
Could this be a `map`, rather than `each_with_object`, as you are constructing another array?

Reply:
Good point 😀 - fixed